topi/python/topi/mali/conv2d.py

```diff
@@ -352,9 +352,10 @@ def _decl_winograd(cfg, data, kernel, strides, padding, dilation, layout, out_dt
     # unpack output
     output = tvm.compute((N, CO, H, W), lambda n, co, h, w:
                          Y[co][n * nH * nW + (h//m) * nW + w//m][h % m][w % m]
-                         # thw following term is used to make the padding effective,
-                         # otherwise the padding will be eliminated by bound inference
-                         + tvm.const(0, out_dtype) * M[alpha-1][alpha-1][CO-1][P_round-1],
+                         # thw following hack term is used to make the padding in batch gemm ("M")
+                         # effective, otherwise the padding will be eliminated by bound inference
+                         + tvm.expr.Mul(tvm.const(0, out_dtype),
```

Comment:
I suggest to leave a comment point to the issue #3088 so ppl understand why `Mul` instead of `*`

Reply:
I'm still confused at why we need this multiplication?

Reply:
thw -> the

Reply:
@icemelon9  During batch gemm, we introduce some padding to avoid partial tile, so we can safely vectorize the innermost loop. However, we won't use all the output of batch gemm (the padded part will be ignored). The InferBound pass in tvm analyzes computation region from output to input, and only keeps the necessary part. If we don't add this term, the padding added in batch gemm will be ignored.

Reply:
@yzliu `tvm.expr.Mul` wont do const fold, while `*` is equal to `tvm.expr.Mul` + const fold.

Reply:
Could you elaborate in the comment by what you replied to @icemelon9 ?

Reply:
It's too long to be put in the comment..